docs/spec/channel.md

```diff
@@ -1,13 +1,14 @@
-# Channel Spec
+# Knative Channel Specification
 
 ## Background
 
-Starting with Version 0.7.0 all the different Channel CRDs (e.g.
-`InMemoryChannel` or `KafkaChannel`) are living in the
-`messaging.knative.dev/v1alpha1` API Group.
+The Knative Eventing project has one generic `Channel` CRD and might ship
+different Channel CRDs implementations (e.g.`InMemoryChannel`) inside of
+in the `messaging.knative.dev/v1beta1` API Group. The generic `Channel` CRD
+points to the chosen _default_ `Channel` implementation, like the `InMemoryChannel`.
 
-A channel logically receives events on its input domain and forwards them to its
-subscribers. Below is a specification for the generic parts of each _Channel_.
+A _channel_ logically receives events on its input domain and forwards them to its
+subscribers. Below is a specification for the generic parts of each `Channel`.
 
 A typical channel consists of a _Controller_ and a _Dispatcher_ pod.
```

Comment:
we probably want to break down the Dispatcher in two logical parts, the receiver and the dispatcher, to make it clear the dispatcher is not only about dispatching events.

Reply:
@lionelvillard fair point, I will get a section in a different PR